framework/src/problems/FEProblemBase.C

```diff
@@ -292,6 +292,10 @@ FEProblemBase::validParams()
       "allow_invalid_solution",
       false,
       "Set to true to allow convergence even though the solution has been marked as 'invalid'");
+  params.addParam<bool>(
+      "show_invalid_solution_console",
+      true,
+      "Set to true to allow convergence but still show the invalid solution occurance in console");
```

Comment:
```suggestion
      "Whether to show the invalid solution occurrence in console");
```
I think it s better to separate the choices between the booleans.
Other does: accept / not the invalid solution
this one does: print / not the solution invalidity table

Reply:
It's already there
`  params.addParam<bool>(
      "allow_invalid_solution",
      false,
      "Set to true to allow convergence even though the solution has been marked as 'invalid'");
  params.addParam<bool>(
      "show_invalid_solution_console",
      true,
      "Set to true to allow convergence but still show the invalid solution occurance in console");`